Tests/Tests/STPAddCardViewControllerTest.m

```diff
@@ -41,6 +42,53 @@ - (STPAddCardViewController *)buildAddCardViewController {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Warc-performSelector-leaks"
 
+- (void)testPrefilledBillingAddress_removeAddress {
```

Comment:
Do these need to be inside the `#pragma clang diagnostic ignored "-Warc-performSelector-leaks"` block? It might be slightly better to move them outside.